mantidproject/mantid
# Enables Replacing window and Waterfall with multi-run mode

fixes #14589

This branch will enables The `Plot Data Representation` combo-box `Replacing Plots` and `Waterfall` to work with multi-focusing. 

**To Test**
`Interface/Diffraction/Engineering Diffraction/Focus`
_Get initial settings from [here](https://github.com/mantidproject/mantid/pull/13501)_

Provide any of three focusing text field with a multiple run number for example `228061-228063`
You can find `ENGINX` inside the system & unit test data
One or various files can be renamed to `228061`, `228062` and `228063`
Make sure the `Plot Focused Workspace` is checked
Select from `Plot Data Representation` combo-box: 
`One Window - Replacing Plots` or `One Window - Waterfall`

When `Focus` clicked;
 If `One Window - Replacing Plots` selected

> it should replace old graph with new graph within one window 

 If `One Window - Waterfall` selected

> it should display a waterfall of all graphs within one window
